README.md

```diff
@@ -1103,6 +1106,28 @@ Toegepast op bovenstaand error model kan onderstaande structuur gehanteerd worde
 
 Aangezien het principe van exception shielding wordt gehanteerd wordt slechts 1 type technical error gedefinieerd, waarbij we de technische error details bovendien verbergen. De identificatie van de technische fout gebeurt steeds aan de hand van de meegegeven identifier. De API consumer dient deze identifier te gebruiken in alle communicatie rond de eigenlijke fout, zodat de operationele teams deze kunnen gebruiken om meer gedetailleerde informatie op te zoeken.
 
+In sommige gevallen kan het nuttig zijn om **extra gegevens** mee te geven zodat de consumer beter begrijpt wat het probleem is, zoals bvb bij validatiefouten :
+
+```json
+{
+"type": "http://api-gateway/digipolis/payment/v1/payments/validation-error",
+"title": "There are validation errors.",
+"status": 400,
+"identifier": "f4D27A4A-6bed-47D6-9487-1961DBB6C631",
+"code": "VAL001",
+"validation-errors": [
```

Comment:
"extraInfo":{"validationErrors": .... : vast attribuut voorzien in root, camelCase gebruiken voor keys